CONTRIBUTING.md

```diff
@@ -0,0 +1,145 @@
+- **Opinião política** - Mantenha discussões técnicas
+- **Filiação sindical** - Informação desnecessária para discussões técnicas
+- **Dados de saúde** - Nunca compartilhe condições médicas específicas
+- **Vida sexual** - Informação irrelevante para o contexto comunitário
+- **Dados genéticos ou biométricos** - Nunca compartilhe
+
+### Dados Pessoais Comuns (que requerem cuidado)
+- **CPF, RG, ou outros documentos** - Nunca compartilhe números completos
+- **Endereço residencial completo** - Use apenas cidade/região quando necessário
+- **Telefone pessoal** - Prefira meios de contato profissionais
+- **E-mail pessoal** - Use e-mail profissional quando apropriado
+- **Dados bancários** - Nunca compartilhe
+- **Senhas ou chaves de API** - Use variáveis de ambiente e exemplos genéricos
+
+## ✅ Boas Práticas para Proteção de Dados
+
+### 1. Anonimização e Pseudonimização
+```
+❌ Evite: "João Silva (CPF: 123.456.789-00) da empresa XYZ reportou este problema"
+✅ Prefira: "Um desenvolvedor da comunidade reportou este problema"
+✅ Ou: "João da empresa XYZ reportou este problema" (apenas nome)
+```
+
+### 2. Compartilhamento de Código e Exemplos
+```
+❌ Evite dados reais:
+API_KEY = "sk-1234567890abcdef"
+USER_EMAIL = "joao.silva@email.com"
+DATABASE_URL = "postgresql://user:pass@real-server.com/db"
+
+✅ Use dados fictícios:
+API_KEY = "sk-your-api-key-here"
+USER_EMAIL = "usuario@exemplo.com"
+DATABASE_URL = "postgresql://usuario:senha@localhost/nome_db"
+```
+
+### 3. Relatórios de Problemas (Issues/Discussions)
+- Use dados fictícios ou anonimizados em exemplos
+- Remova logs que contenham informações pessoais
+- Substitua informações sensíveis por placeholders
+
+### 4. Screenshots e Imagens
+- Borre ou edite informações pessoais visíveis
+- Use dados de teste em demonstrações
+- Evite capturar informações da barra de endereços que possam conter dados sensíveis
+
+## 🔒 Segurança em Repositórios
+
+### 1. Secrets e Variáveis de Ambiente
+- Nunca faça commit de arquivos `.env` com dados reais
+- Use `.env.example` com valores de exemplo
+- Configure adequadamente o `.gitignore`
+
+### 2. Historico do Git
+- Verifique commits antes de enviar
+- Use `git log --oneline` para revisar histórico
+- Se dados sensíveis foram commitados acidentalmente, remova do histórico
+
+### 3. Configurações e Logs
+- Sanitize logs antes de compartilhar
+- Remova tokens de autenticação de configurações de exemplo
+- Use ferramentas como `git-secrets` para prevenção
+
+## 📋 Checklist de Verificação
+
+Antes de postar ou fazer commit, verifique:
+
+- [ ] Não há CPF, RG, ou outros documentos pessoais
+- [ ] Endereços e telefones foram removidos ou anonimizados
+- [ ] E-mails pessoais foram substituídos por exemplos genéricos
+- [ ] Credenciais de API e senhas estão como placeholders
+- [ ] Screenshots não contêm dados pessoais
+- [ ] Logs foram sanitizados
+- [ ] Dados de exemplo são fictícios
+- [ ] Informações sensíveis sobre terceiros foram removidas
```

Comment:
Se for adicionar outras boas práticas, esse checklist deveria ter outros pontos além de LGPD. E acho que alguns pontos aqui remetem muito a código, e esse repositório não é um repositório de código e sim de discussões.

Reply:
Seção removida pois não se aplica ao contexto de fórum.